ref_builder/cli.py

```diff
@@ -214,6 +214,17 @@ def otu_promote_accessions(
     type=str,
     required=True,
 )
+@click.option(
+    "--ignore-name",
```

Comment:
```suggestion
    "--ignore-genbank-name",
```
---
Is this going to result in an unnamed isolate, then? In that case, I think the option should be `--unnamed`.